client/verta/verta/_internal_utils/_utils.py

```diff
@@ -265,9 +265,19 @@ def _get_visible_orgs(self):
         )
         response = self.must_proto_response(response, Workspace_pb2.Workspaces)
 
-        org_names = map(lambda workspace: workspace.org_name, response.workspace)
-        org_names = filter(None, org_names)
-        return list(org_names)
+        workspace_names = map(lambda workspace: "{}:{}".format(workspace.org_id, workspace.org_name), response.workspace)
+        workspace_names = filter(None, workspace_names)
+        return list(workspace_names)
```

Comment:
Could you please help rename this function to `_get_visible_workspaces()`? 🙇‍♂️

Reply:
renamed. This change was in personal workspace removal PR, but can add it here too